examples/hellohttp/e2e-test.sh

```diff
@@ -42,20 +42,38 @@ function EXPECT_CONTAINS() {
 }
 
 if [[ -z "${1:-}" ]]; then
-  echo "Usage: $(basename $0) <kubernetes namespace> <language ...>"
+  echo "Usage: $(basename $0) <'remote' or 'local' run> <kubernetes namespace> <language ...>"
+  exit 1
+fi
+
+local=false
+if [[ "$1" == "local" ]]; then
+  local=true
+elif [[ "$1" != "remote" ]]; then
+  echo "Usage: $(basename $0) <'remote' or 'local' run> <kubernetes namespace> <language ...>"
+  exit 1
+fi
+shift
+
+if [[ -z "${1:-}" ]]; then
+  echo "Usage: $(basename $0) <'remote' or 'local' run> <kubernetes namespace> <language ...>"
   exit 1
 fi
 namespace="$1"
 shift
 if [[ -z "${1:-}" ]]; then
   echo "ERROR: Languages not provided!"
-  echo "Usage: $(basename $0) <kubernetes namespace> <language ...>"
+  echo "Usage: $(basename $0) <'remote' or 'local' run> <kubernetes namespace> <language ...>"
   exit 1
 fi
 
 get_lb_ip() {
-    kubectl --namespace="${namespace}" get service hello-http-staging \
-      -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+  ip_var='{.status.loadBalancer.ingress[0].ip}'
```

Comment:
add doc comment about what you are doing here

Reply:
Done